WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletInfoViewModel.cs

```diff
@@ -0,0 +1,65 @@
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Userfacing;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Advanced
+{
+	[NavigationMetaData(Title = "Wallet Info")]
+	public partial class WalletInfoViewModel : RoutableViewModel
+	{
+		[AutoNotify] private bool _showSensitiveData;
+		[AutoNotify] private string _showButtonText = "Show sensitive data";
+		[AutoNotify] private string _lockIconString = "eye_show_regular";
+
+		public WalletInfoViewModel(Wallet wallet)
+		{
+			var network = wallet.Network;
+
+			SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+
+			EnableCancel = !wallet.KeyManager.IsWatchOnly;
+
+			NextCommand = ReactiveCommand.Create(() => Navigate().Clear());
+
+			CancelCommand = ReactiveCommand.Create(() =>
```

Comment:
I dont understand cancel in this context.

Reply:
The bottom left button is not `Cancel` here, It the `Show sensitive data`.
![image](https://user-images.githubusercontent.com/16364053/118967870-83defb80-b96b-11eb-8d1c-6f46b03ac8f4.png)

I would refactor the names in `RoutableViewModel`...
A solution can be 
- CancelCommand -> LeftButtonCommand
- NextCommand -> RightButtonCommand

Also, the NextCommand in several cases is not Next. (Done, OK, Finish)

Reply:
ah I see! ok